### DISCUSSION TITLE
Loop up and down

### DISCUSSION THREAD

**Discussion by omnigenous** [gokcehan/lf#1734] [Q&A] [answered]
```
file1
file2
file3 <--- cursor
```

If you press `down` in this state nothing will happen. Is to possible to make it so when you press `down`, in this state, cursor moves to `file1`?

---

:tada:  `down`

```
file1 <--- cursor
file2
file3
```

**Comment by DusanLesan** [comment, marked as answer]
To achieve that, you need to set [wrapscroll](https://github.com/gokcehan/lf/blob/master/doc.md#wrapscroll-bool-default-false)
```
set wrapscroll true
```
